src/video/wayland/SDL_waylanddyn.c

```diff
@@ -28,6 +28,45 @@
 
 #ifdef SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC
 
+SDL_ELF_NOTE_DLOPEN(
+    "wayland",
+    "Support for Wayland video",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_SUGGESTED,
+    SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC
+);
+#ifdef SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC_EGL
+SDL_ELF_NOTE_DLOPEN(
+    "wayland-egl",
+    "Support for Wayland video",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_SUGGESTED,
+    SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC_EGL
+);
+#endif
+#ifdef SDL_VIDEO_DRIVER_WAYLAND_DYNAMIC_CURSOR
+SDL_ELF_NOTE_DLOPEN(
+    "wayland-cursor",
```

Comment:
Should these (and the rest of the Wayland libraries) all be grouped as a `"wayland"` topic?